index.html

```diff
@@ -809,6 +850,7 @@ <h2>DisplayMediaStreamConstraints</h2>
   (boolean or MediaTrackConstraints) audio = false;
   SelfCapturePreferenceEnum selfBrowserSurface;
   SystemAudioPreferenceEnum systemAudio;
+  DynamicSourceOptionPreferenceEnum dynamicSourceOption;
```

Comment:
I find the name cryptic. Can we name it canChangeCapturedSurface?

Reply:
I'm OK with whichever name @jan-ivar and @youennf will agree to. Including this one.

Reply:
I like Harald's suggestion.

Reply:
`surfaceSwitching: "yes" | "no"`

Reply:
Actually, I like "include" and "exclude" here since it's consistent with recent enums we've added around surface inclusion/exclusion. This is slightly different since it's a dynamic feature to switch surfaces later, but seems similar in that it has to do with what surfaces the user can choose.

Those other enums were also hints, so being consistent in naming with them would be nice, and might help readers intuitively understand they're all hints